themebuilder/modules/commands.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     BUILD_THEME: "build-theme",
-    BUILD_VARS: "build-theme-vars"
+    BUILD_VARS: "build-theme-vars",
```

Comment:
Not sure about "build-theme-vars" and "build-theme-meta" commands. In my opinion, "get-theme-cars" and "get-theme-meta" looks more understandable. I want us to divide the main command from the additional ones more.

@dxbykov what do you think?

Reply:
As for me, 'get' sounds like it returns something (into the console), not writes a file. As an alternative to 'build', I can suggest 'export' or 'create'.